frontend/lib/src/components/widgets/CustomComponent/ComponentRegistry.ts

```diff
@@ -64,6 +64,23 @@ export class ComponentRegistry {
     }
   }
 
+  public checkSourceResponse = (
+    source: string,
+    componentName?: string
+  ): void => {
+    this.endpoints.checkSourceResponse(source, componentName)
```

Comment:
nit: Seems like this should just be returned for simplicity to mirror the endpoints function and to allow further behavior (perhaps for testing purposes). Certainly don't need to utilize it though.